AddShippingZone.tsx

```diff
@@ -25,8 +26,20 @@ const AddShippingZone = () => {
           onPress: () => console.log("Toolbar action button pressed"),
         }),
     });
+    BackHandler.addEventListener("hardwareBackPress", handleBackButtonClick);
+    return () => {
+      BackHandler.removeEventListener(
+        "hardwareBackPress",
+        handleBackButtonClick
+      );
+    };
   }, [navigation]);
```

Comment:
This does not seem to have any effect on iOS, which is good! But I wonder, do we have to add this handle on each new screen?